OpenRA.Game/Map/Map.cs

```diff
@@ -285,6 +287,26 @@ public static string ComputeUID(IReadOnlyPackage package)
 			}
 		}
 
+		// PERF This is a way to get MapFormat without expensive yaml parsing
```

Comment:
Please put this before the `Regex.Match` line instead.

Reply:
ok